apps/package.json

```diff
@@ -253,8 +168,20 @@
     "@blockly/plugin-scroll-options": "~5.0.8",
```

Comment:
Starting here we're in the dependencies section of package.json, red here means I moved something TO devDeps FROM deps